.github/workflows/ci.yml

```diff
@@ -130,7 +130,7 @@ jobs:
       - name: Install dependencies
         run: npm ci
       - name: E2E test affected projects
-        run: npx nx affected:e2e --parallel=3
+        run: npx nx affected:e2e --parallel=1
```

Comment:
@matejchalk I had to run the e2e tests in parallel to avoid a bug while publishing

Reply:
What was the bug? Was it related to all E2E or just the Lighthouse E2E? It would be very inconvenient to lose the parallel execution of E2E tests.

Reply:
I assume I have a clue now. The plugins are not executed in sequence. 

Will get back with a test case...

Reply:
Here a screenshot of console.logs... 

I putted logs inside the runner function (which in any case should execute sequential)

In the screen shot you see that first all runner log, then the content of the first runner logs.

<img width="386" alt="Screenshot 2024-02-28 at 14 34 53" src="https://github.com/code-pushup/cli/assets/10064416/51342ac0-8319-4388-9e49-ec9f3fdbe9db">

Reply:
> I had to run the e2e tests in parallel to avoid a bug while publishing. Is this related to the bug you solved recently?

No clue what your bug is, but the bug I fixed was about `Cannot use import statement outside a module` errors in #493.